app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
I'm still 👎  having an engine provide `can :manage, :all`; it's obnoxious for an application to undo, and requires overriders to closely track implicit ability changes. I'm fine with `:manage, [list of models we provide/care about]`, though.

Having an explicit admin role in the rights metadata seems appropriate, although I wonder if it'll get us in trouble elsewhere (e.g. needing to disallow other managers from removing the admin group). Or, maybe there's something more subtle going on (e.g. "application admins should be able to manage any object created through the application").